model/edx.go

```diff
@@ -100,9 +94,29 @@ func (e *EdxLMS) BuildUser(launchData map[string]string, password string) (*User
 		return nil, NewAppError("Edx_BuildUser", "edx.build_user.lti_user_id_missing", nil, "", http.StatusBadRequest)
 	}
 
+	firstName := strings.Trim(launchData[launchDataFirstNameKey], " ")
+	lastName := strings.Trim(launchData[launchDataLastNameKey], " ")
+
+	if firstName == "" || lastName == "" {
+		name := strings.Split(strings.Trim(launchData[launchDataFullNameKey], " "), " ")
+		if len(name) > 0 {
+			if firstName == "" {
+				firstName = name[0]
+			}
+
+			if lastName == "" {
+				lastName = strings.Join(name[1:], " ")
```

Comment:
Can we use `strings.SplitN` here instead?

Reply:
Done.